### ISSUE TITLE
Accessing closed database at shutdown

### ISSUE THREAD

**Issue by AlexandraRoatis** [aionnetwork/aion#205]
Version [0.1.15.ec711be](https://github.com/aionnetwork/aion/commit/ec711be0c4239d6961cff154fa2f23635db6c414)

```
18-03-16 14:46:48.632 INFO  GEN  [Shutdown]: shutting down consensus...
18-03-16 14:46:48.632 INFO  GEN  [Shutdown]: shutdown consensus... Done!
18-03-16 14:46:48.632 INFO  GEN  [Shutdown]: shutting down DB...
Exception in thread "sync-ib" java.lang.RuntimeException: Database is not opened: LevelDBWithCache:<name=details,autocommit=OFF,size=UNBOUND,stats=ON>
	at org.aion.db.impl.AbstractDatabaseWithCache.check(Unknown Source)
	at org.aion.db.impl.AbstractDatabaseWithCache.keys(Unknown Source)
	at org.aion.mcf.db.DetailsDataStore.syncLargeStorage(Unknown Source)
	at org.aion.zero.impl.db.AionRepositoryImpl.commitBlock(AionRepositoryImpl.java:456)
	at org.aion.zero.impl.AionBlockchainImpl.storeBlock(AionBlockchainImpl.java:925)
	at org.aion.zero.impl.AionBlockchainImpl.add(AionBlockchainImpl.java:553)
	at org.aion.zero.impl.AionBlockchainImpl.tryToConnect(AionBlockchainImpl.java:440)
	at org.aion.zero.impl.sync.TaskImportBlocks.run(TaskImportBlocks.java:94)
	at java.base/java.lang.Thread.run(Thread.java:844)
18-03-16 14:46:48.647 INFO  GEN  [Shutdown]: Details data source closed.
18-03-16 14:46:48.660 INFO  GEN  [Shutdown]: State database closed.
18-03-16 14:46:48.660 INFO  GEN  [Shutdown]: Transaction database closed.
18-03-16 14:46:48.663 INFO  GEN  [Shutdown]: Block store closed.
18-03-16 14:46:48.663 INFO  GEN  [Shutdown]: shutdown DB... Done!
```

**Comment by AlexandraRoatis** [comment]
The thread that attempted the database access:
https://github.com/aionnetwork/aion/blob/63d58d43fa214aeeab0450e400e15f56db6625dc/modAionImpl/src/org/aion/zero/impl/sync/SyncMgr.java#L164

**Comment by yulongaion** [comment]
Fixed by b100f84162af4e1a8a8e7f085aadedc30e1f3739
